pkg/reconciler/pipelinerun/affinity_assistant_test.go

```diff
@@ -1157,14 +1154,12 @@ func TestCleanupAffinityAssistantsAndPVCs_Failure(t *testing.T) {
 	}
 }
 
-// TestThatCleanupIsAvoidedIfAssistantIsDisabled tests that
-// cleanup of Affinity Assistants is omitted when the
-// Affinity Assistant is disabled
-func TestThatCleanupIsAvoidedIfAssistantIsDisabled(t *testing.T) {
+// TestThatCleanupIsAvoidedtests that cleanup of Affinity Assistants is omitted
+func TestThatCleanupIsAvoided(t *testing.T) {
 	configMap := &corev1.ConfigMap{
 		ObjectMeta: metav1.ObjectMeta{Name: config.GetFeatureFlagsConfigName(), Namespace: system.Namespace()},
 		Data: map[string]string{
-			featureFlagDisableAffinityAssistantKey: "true",
+			"coschedule": "disabled",
```

Comment:
As mentioned in the action required, if you want to maintain the execution behavior, you may need to set the `coschedule` to disabled.

The focus is on the line of code below: (line: 53)
https://github.com/tektoncd/pipeline/blob/aec465ee5898db53bbf26320408d40dca6e6e446/pkg/internal/affinityassistant/affinityassistant_types.go#L34-L55